http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/GKEInterpreter.scala

```diff
@@ -1596,6 +1704,61 @@ class GKEInterpreter[F[_]](
     ) ++ configs ++ galaxyRestoreSettings
   }
 
+  private[util] def buildRStudioAppChartOverrideValuesString(
+    appName: AppName,
+    cluster: KubernetesCluster,
+    nodepoolName: NodepoolName,
+    namespaceName: NamespaceName,
+    disk: PersistentDisk,
+    ksaName: ServiceAccountName,
+    userEmail: WorkbenchEmail,
+    stagingBucket: GcsBucketName
+  ): List[String] = {
+    val rstudioIngressPath = s"/proxy/google/v1/apps/${cluster.cloudContext.asString}/${appName.value}/rstudio-service"
+    val welderIngressPath = s"/proxy/google/v1/apps/${cluster.cloudContext.asString}/${appName.value}/welder-service"
+    val k8sProxyHost = kubernetesProxyHost(cluster, config.proxyConfig.proxyDomain).address
+    val leoProxyhost = config.proxyConfig.getProxyServerHostName
+
+    val rewriteTarget = "$2"
+    val ingress = List(
+      raw"""ingress.enabled=true""",
+      raw"""ingress.annotations.nginx\.ingress\.kubernetes\.io/auth-tls-secret=${namespaceName.value}/ca-secret""",
+      raw"""ingress.annotations.nginx\.ingress\.kubernetes\.io/proxy-redirect-from=https://${k8sProxyHost}""",
+      raw"""ingress.annotations.nginx\.ingress\.kubernetes\.io/proxy-redirect-to=${leoProxyhost}""",
+      raw"""ingress.annotations.nginx\.ingress\.kubernetes\.io/rewrite-target=/${rewriteTarget}""",
+      raw"""ingress.hosts[0].host=${k8sProxyHost}""",
+      raw"""ingress.hosts[0].paths[0].path=${rstudioIngressPath}${"(/|$)(.*)"}""",
+      raw"""ingress.hosts[0].paths[0].backend.service.name=aou-rstudio""",
+      raw"""ingress.hosts[0].paths[0].backend.service.port.number=8787""",
```

Comment:
should `service.name, service.port.number` provided by Helm so that it can match the port/service name provided in `deployment.yam`, and `svc.yaml`?

Reply:
i could not find a way of gracefully do that though given that we are getting the path from leo within this loop

Reply:
It might be nice if the `ingress.yaml` in the helm chart were a little less templated, so Leo didn't have to pass these hardcoded values..

Reply:
Okay, let me have a second pass at the`ingress.yaml` file

Reply:
I think I greatly simplified the ingress yaml file, see https://github.com/DataBiosphere/terra-app/pull/34/commits/389f058075ecbd516ef722d86dfeb787194a371b

Everything is still working great on my BEE, but I am hesitant merging this PR around 5pm ET on a Friday before a long weekend in case nobody is around to help me revert it so I might wait until Monday evening / Tuesday morning 😬